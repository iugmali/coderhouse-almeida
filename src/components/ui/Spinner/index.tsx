import {twMerge} from "tailwind-merge";

const Spinner = ({className}: {className?: string}) => {
  return (
    <div className={twMerge(`rounded-full w-16 h-16 border-gray-200 border-4 border-t-blue-500 border-l-blue-500 animate-spin`, className)} />
  )
}

export default Spinner;
