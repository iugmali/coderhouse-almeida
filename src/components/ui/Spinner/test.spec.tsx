import {render} from "@testing-library/react";
import Spinner from "@/components/ui/Spinner/index";

describe('Spinner', () => {
  it('renders unchanged', () => {
    const {container} = render(<Spinner />)
    expect(container.firstChild).toMatchSnapshot()
  })
})
