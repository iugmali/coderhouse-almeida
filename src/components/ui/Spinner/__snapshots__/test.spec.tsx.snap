// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner renders unchanged 1`] = `
<div
  class="rounded-full w-16 h-16 border-gray-200 border-4 border-t-blue-500 border-l-blue-500 animate-spin"
/>
`;
